src/System.CommandLine/src/System/CommandLine/Argument.cs

```diff
@@ -38,6 +39,8 @@ public bool IsParameter
 
         public bool IsHidden { get; set; }
 
+        public bool RequireValue { get; private set; }
```

Comment:
@drewnoakes Agreed. I'll make the change if/when I rebase this. The PR is a bit stale and I've been waiting for some sign from the maintainers that they're interested in the change before spending the time to rebase.